libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -650,3 +653,28 @@ def _prepare_chat(
         message = history.pop()
         chat = self.client.start_chat(history=history)
         return params, chat, message
+    
+    @property
+    def is_gemini(self) -> bool:
```

Comment:
I don't think we want it to be parameterized with a bool, if there will be multiple families of models, we want to return the family identifier rather than a false / true.

Perhaps better to avoid creating a property or else marking it as private?

Reply:
@eyurtsev Add _get_model_family() to retreive model family